templates/Program/program_description.html.twig

```diff
@@ -97,10 +151,59 @@
             </textarea>          </span>
           <span class="mdc-line-ripple"></span>
         </label>
-        <button id="edit-credits-submit-button" class="btn btn-primary">
-          <i class="material-icons align-bottom">save</i>
-          {{ 'programs.saveCredits'|trans({}, 'catroweb') }}
-        </button>
+
+        <div class="row justify-content-between" style="margin-top: 1rem; padding-right: 0.75rem;">
+          <div class="form-group col-8">
+            <div id="credits-language-selector" class="mdc-select mdc-select--outlined"  {% if (program.credits == null) %} style="display: none;" {% endif %}>
+              <div class="mdc-select__anchor" aria-labelledby="outlined-select-label">
+                <span class="mdc-notched-outline">
+                  <span class="mdc-notched-outline__leading"></span>
+                  <span class="mdc-notched-outline__notch">
+                    <span id="credits-outlined-select-label"
+                          class="mdc-floating-label">{{ 'footer.language'|trans({}, "catroweb") }}</span>
+                  </span>
+                  <span class="mdc-notched-outline__trailing"></span>
+                </span>
+                <span class="mdc-select__selected-text-container">
+                  <span id="credits-selected-text" class="mdc-select__selected-text" data-selected-value="default"></span>
+                </span>
+                <span class="mdc-select__dropdown-icon">
+                  <svg
```

Comment:
can we do some quick refactor, like extracting this svg to a separate file?

Reply:
Just the svg section or were you thinking of breaking it up a little more?

Reply:
the svg would be a good start

Reply:
Okay, I moved the svg section out into its own file